ocular-dev-tools.config.js

```diff
@@ -22,7 +22,10 @@ const config = {
   },
 
   aliases: {
-    'deck.gl-test': resolve(__dirname, './test')
+    'deck.gl-test': resolve(__dirname, './test'),
+    '@loaders.gl/loader-utils': `${__dirname}/../loaders.gl/modules/loader-utils/src`,
+    '@loaders.gl/mvt': `${__dirname}/../loaders.gl/modules/mvt/src`,
+    '@loaders.gl/gis': `${__dirname}/../loaders.gl/modules/gis/src`
```

Comment:
Maybe use an environment variable for the `loaders.gl` repo location? It's not guaranteed to be in the same directory as `deck.gl`

Reply:
I added this only for debugging purpose on my local dev... I wanted to delete this when moving the PR out the draft. Do you think will be interested to improve this?

Reply:
I think it's fine to leave this out completely